add-on/chrome/modules/LoopRooms.jsm

```diff
@@ -746,9 +746,26 @@ var LoopRoomsInternal = {
       eventEmitter.emit("close");
     });*/
 
-    let win = Services.wm.getMostRecentWindow("navigator:browser");
-    win.LoopUI.loadSidebar(roomToken);
-    MozLoopService.openURL(TOC_URL + roomToken);
+    let urlToLoad =
+      Cc["@mozilla.org/supports-string;1"].createInstance(Ci.nsISupportsString);
+    urlToLoad.data = TOC_URL + roomToken;
+
+    let windowArgs =
+      Cc["@mozilla.org/supports-array;1"].createInstance(Ci.nsISupportsArray);
+    windowArgs.AppendElement(urlToLoad);
+
+    // XXX Do we need to do anything more sophisticated (eg handle private
+    // browsing mode)?  If so, we may want to use utilityOverlay.js:openUILinkIn
+    // (which calls utilityOverlay.js:openLinkIn).  Doing this is rather gross,
+    // as we would need to get the most recently used window and then reach in
+    // and invoke window.openUILinkIn, and it's also conceivably fragile, since
+    // this is unlikely to be a supported way to use that API.
```

Comment:
Digging through `openLinkIn`, it actually ends up calling `Services.ww.openWindow` as you do here, but with `",private"` on the end of the features.

When I then follow that `OpenWindow` through into nsWindowWatcher.cpp it appears that all the "features" are hard-coded in c++ land. That's going to make it super difficult for us to have a specific type of window.

Since for various things we need to be able to detect if a window is our own or not, what we could do is when a window is opened, and `WindowListener#onOpenWindow` (in bootstrap) detects it, then we can check the url being loaded, if its `about:looptoc` then we can set a flag on the window, we can also potentially use that to add our extra UI.

We don't have to do that all here, but I'm thinking that may be a better thing to start to work towards.